### DISCUSSION TITLE
Error generateWSALinks.py", line 32, in <module>

### DISCUSSION THREAD

**Discussion by christophe-lenglet** [LSPosed/MagiskOnWSALocal#474] [General]
![image](https://user-images.githubusercontent.com/108875922/229490392-d80950f1-dc69-4eda-9431-07cb2997e8f8.png)

COMMAND_LINE=--arch x64 --release-type retail --magisk-ver stable --gapps-brand MindTheGapps --gapps-variant pico  --root-sol magisk     --compress-format
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
Build: RELEASE_TYPE=Retail
Generate Download Links
Traceback (most recent call last):
  File "/mnt/c/users/chris/MagiskOnWSALocal/scripts/generateWSALinks.py", line 32, in <module>
    from requests import Session
ModuleNotFoundError: No module named 'requests'
Build: an error has occurred, exit

Cleanup Work Directory

Please Help Me !!! Thank You

**Comment by hitman-snipe** [comment]
I have this problem too. and I don't see the GApps variants selection to choose.

**Comment by tinycows** [comment]
i had this problem. says rquests is not installed. solved by installing requests.  sudo apt-get install python3-requests

but now i have a new error: ./build.sh: line 437: aria2c: command not found
We have encountered an error while downloading files.

**Reply by TheNoob21** [reply]
Please update if you find a solution

**Reply by TheNoob21** [reply]
This worked for me: 

Update Ubuntu and install git: sudo apt update && sudo apt install git

**Comment by tinycows** [comment]
what worked for me was to install aria2c, after that everything worked
